README.md

```diff
@@ -73,19 +73,8 @@ flutter doctor -v
 
 If you only plan to work on the `web` target we recommend installing Google Chrome as it is the default supported target *(Flutter being developped by Google)*.
 
-It is also **required** to have a web ready version of libolm available in the `assets/js/package` folder. You can build a version using:
+It is also **required** to have a web ready version of libolm available in the `assets/js/package` folder. You can download it [here](https://gitlab.matrix.org/matrix-org/olm/-/packages/2190) and copy olm.js and olm.wasm in `assets/js/package`.
```

Comment:
IMO, we should keep both guideline for user.

Reply:
I can add it so we have both. But is there any good reason to the Docker way? Even if I have no background to this library, I do not see the purpose of building it locally if its a web library?

Reply:
Why close @hoangdat ? Keeping both can be a safe solution, but it was easier and it worked on my local setup by just downloading it from the link. I can create a new PR with both solution.